storage/tianmu/types/tianmu_value_object.cpp

```diff
@@ -0,0 +1,165 @@
+/* Copyright (c) 2022 StoneAtom, Inc. All rights reserved.
```

Comment:
Why the left file is empty, We just rename rc_value_object.cpp to tainmu_value_ojbect.cpp. I remeber git can deal this ,but I can not point it out the detailes.

Reply:
This situation is that the file had been reformated from windows to unix.